staging/src/k8s.io/kubectl/pkg/cmd/certificates/certificates.go

```diff
@@ -146,11 +146,14 @@ func NewCmdCertificateApprove(f cmdutil.Factory, ioStreams genericclioptions.IOS
 }
 
 func (o *CertificateOptions) RunCertificateApprove(force bool) error {
-	return o.modifyCertificateCondition(o.builder, o.clientSet, force, func(csr *certificatesv1beta1.CertificateSigningRequest) (*certificatesv1beta1.CertificateSigningRequest, bool) {
+	return o.modifyCertificateCondition(o.builder, o.clientSet, force, func(csr *certificatesv1beta1.CertificateSigningRequest, force bool) (*certificatesv1beta1.CertificateSigningRequest, bool) {
 		var alreadyApproved bool
-		for _, c := range csr.Status.Conditions {
-			if c.Type == certificatesv1beta1.CertificateApproved {
+		for i := range csr.Status.Conditions {
+			if csr.Status.Conditions[i].Type == certificatesv1beta1.CertificateApproved {
 				alreadyApproved = true
+				if force {
+					csr.Status.Conditions[i].LastUpdateTime = metav1.Now()
```

Comment:
I wouldn't expect this… we're not changing the condition so we shouldn't change the timestamp

Reply:
When approve|deny --force, kubectl will request the apiserver to make changes

![image](https://user-images.githubusercontent.com/7907809/83982678-1654dd80-a95b-11ea-8504-8d67ee4cd785.png)

So, I think it makes sense to update the LastUpdateTime of the condition at this time